src/Equinor.ProCoSys.BusReceiver/Topics/ProjectTopic.cs

```diff
@@ -5,6 +5,7 @@ public class ProjectTopic
         public string ProjectSchema { get; set; }
         public string ProjectName { get; set; }
         public string Description { get; set; }
+        public string IsClosed { get; set; }

```

Comment:
Should we have a project table is IPO as in Preservation where we can sync this? This is what I wanted to discuss with you, JI and Lykke

Reply:
Not relevant for this task, but just seeing places where it would make sense

Reply:
From the very beginning of IPO, I found it "strange" that you decided to not have a Project-table. IPO as it is, do not handle the truth that a project can be closed. (Data belonging to a closed project should nor be changed). And, as you've seen now, a move of CommPkg from a project to another can cause a "mass update" of CommPkg/McPkg rows in IPO

Reply:
It might be relevant later. But we made a decision in the beginning to keep the structure as flat as possible, so we did not want a project table, as it is not necessary for now.